modules/rostests/rosautotest/main.cpp

```diff
@@ -87,6 +89,12 @@ extern "C" int
 wmain(int argc, wchar_t* argv[])
 {
     int ReturnValue = 1;
+    DWORD TestStartTime, TestEndTime;
+
+    GetModuleFileNameW(NULL, TestName, _countof(TestName));
+    WCHAR* Name = wcsrchr(TestName, '\\');
+    Name++;
```

Comment:
In theory we should test for Name being NULL or not before the '++'.

Reply:
I added an "if (Name)" test as a condition for the "Name++;". Thanks.